tests/data/info/site_info.json

```diff
@@ -416,24 +416,24 @@
   },
   "WAO": {
     "ICOS": {
-      "height": ["21m"],
-      "height_name": ["20magl"],
+      "height": ["10m"],
+      "height_name": {"10m": ["10magl", "20magl"]},
```

Comment:
Should 20magl be here?

Reply:
No, this is a mapping between the inlet height (defined within "height" above) and the potential footprint height values. We want to map the "10m" inlet against both the "10magl" and "20magl" footprints i.e. first look for "10magl" then "20magl" but either can be used.